common-files/Jenkinsfile

```
@@ -0,0 +1,11 @@
+/*
+ See the documentation for more options:
+ https://github.com/jenkins-infra/pipeline-library/
+*/
+def recentLTS = "2.222.1"
+
+buildPlugin(useAci: true, configurations: [
+        [ platform: "linux", jdk: "8", jenkins: null ],
+        [ platform: "windows", jdk: "8", jenkins: recentLTS, javaLevel: "8" ],
+        [ platform: "linux", jdk: "11", jenkins: recentLTS, javaLevel: "8" ],
```

```suggestion
    [platform: 'linux', jdk: '8'],
    [platform: 'windows', jdk: '8', jenkins: recentLTS],
    [platform: 'linux', jdk: '11', jenkins: recentLTS],
```

I am also not sure I would advise these combinations. Mixing Windows with a nondefault `jenkins` complicates diagnosis. So does mixing JDK 11 with a recent LTS version. We do not really have good universal guidance here.

---

sure it was taken from https://github.com/jenkins-infra/pipeline-library/blob/master/vars/buildPlugin.groovy#L268 with a more recent baseline